src/components/dropdown-search/dropdown-search.js

```diff
@@ -0,0 +1,111 @@
+import React, { useState } from "react";
+import PropTypes from "prop-types";
+import "./dropdown-search.scss";
+import ArrowDown from "../../images/icons/arrow-down.svg";
+import { SearchInput } from "../search-input/search-input";
+
+export const DropdownSearch = ({
+  title,
+  options,
+  onSelect,
+  isAlwaysOpen,
+  showSearchInput = true,
+  searchPlaceholder
+}) => {
+  const [showDropdownOptions, setShowDropdownOptions] = useState(false);
+  const [dropdownTitle, setDropdownTitle] = useState(title);
+  const [dropdownOptions, setDropDownOptions] = useState(options);
```

Comment:
Would be nice to call this method `setDropdownOptions` as we don't have `DropDown` in other places 😁

Reply:
Done.